ArjunaCore/arjuna/classes/com/arjuna/ats/arjuna/coordinator/TransactionReaper.java

```diff
@@ -768,6 +757,25 @@ private final void shutdown(boolean waitForTransactions)
         _reaperWorkerThread = null;
     }
 
+    /**
+     * Wait for all the transactions to terminate normally.
+     *
+     * Note: this method assumes that the transaction system has been
+     * shutdown already so no new transactions can be created, or we
```

Comment:
I think this Javadoc should be reworked from what was added originally. I think it should say. `Note: this method assumes that the transaction system is not currently capable of creating new transactions (via TxControl.disable()).` (please can I ask you verify the syntax of the TxControl method I mentioned)

Reply:
Thanks Tom, I will update this javadoc

Reply:
Done, thanks Tom